From dac72f1d7b4ab1f24bdc64bd2b3f96bd0e30e489 Mon Sep 17 00:00:00 2001
From: David Henry <davi.henry@orange.fr>
Date: Fri, 18 Nov 2022 22:28:01 +0000
Subject: [PATCH] Fix missing GtkRadioButton id

Add missing id to custom scale radio button referenced in caja-image-resizer.c
This fixes issue #104.
---
 image-converter/caja-image-resize.ui | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/image-converter/caja-image-resize.ui b/image-converter/caja-image-resize.ui
index 5fe0ea35..f3bcbce4 100644
--- a/image-converter/caja-image-resize.ui
+++ b/image-converter/caja-image-resize.ui
@@ -183,7 +183,7 @@
                             <property name="can_focus">False</property>
                             <property name="spacing">8</property>
                             <child>
-                              <object class="GtkRadioButton">
+                              <object class="GtkRadioButton" id="custom_pct_radiobutton">
                                 <property name="label" translatable="yes">Scale:</property>
                                 <property name="visible">True</property>
                                 <property name="can_focus">True</property>
